src/gei/Commands/GenerateScript/GenerateScriptCommandArgs.cs

```diff
@@ -23,6 +23,7 @@ public class GenerateScriptCommandArgs : CommandArgs
         public string GithubSourcePat { get; set; }
         public bool KeepArchive { get; set; }
         public string TargetApiUrl { get; set; }
+        public bool UseGithubStorage { get; set; }
```

Comment:
The following two validations need to be added now that we have the GitHub storage option:
1. The `UseGithubStorage` is provided but no `GhesApiUrl`
2. `UseGithubStorage` is provided alongside `AwsBucketName`